src/pages/news/2021-03-30-resolving-vulnerabilities-podcast.en.md

```diff
@@ -0,0 +1,20 @@
+---
+path: "/en/news/2021-03-30-resolving-vulnerabilities-podcast/"
+date: "2021-03-30"
+title: "Podcast interview: resolving vulnerabilities in OpenZaak"
+lang: en
+template: default
+---
+
+## Podcast interview: resolving vulnerabilities in OpenZaak
+
+Several weeks ago, OpenZaak was featured in the livecast series ['Let's talk about Public Code'](https://podcast.publiccode.net/) from the [Foundation for Public Code](publiccode.net). The purpose of this show is to share stories of codebases that are working with Public Code from different viewpoints, (developers, vendors, civil servants, managers, etc.).
+
+The episode dedicated to OpenZaak was focused on the developers point of view regarding a [vulnerability](https://github.com/publiccodenet/projects/issues/20) that was discovered, and resolved within the community and what learnings were taken from it. 
+
+The interview with [Sergei Maertens](https://github.com/sergei-maertens), technical lead at [Maykin Media](https://github.com/maykinmedia) and [Tjerk Vaags](https://github.com/TjerkVaags), developer at [Contezza](https://contezza.nl/), is available in different formats:
+
+- Watch it on [Youtube](https://www.youtube.com/watch?v=1xojrumKgfA)
+- [Periscope](https://www.pscp.tv/w/1gqxvoRWjgwKB)
+- [Facebook](https://www.facebook.com/publiccodenet/videos/321846445885858/)
+- Or you can just do the old school thing and listen it in [podcast format](https://podcast.publiccode.net/e/3-sergei-maertens-and-tjerk-vaags-openzaak/).
```

Comment:
consider rephrasing "old school thing" to " or you can listen to it in audio podcast format"